pkg/model/manifest.go

```diff
@@ -491,7 +496,11 @@ func (pf PortForward) ToLink() Link {
 	if host == "" {
 		host = "localhost"
 	}
-	url := fmt.Sprintf("http://%s:%d/", host, pf.LocalPort)
+	path := pf.Path
+	if strings.HasPrefix(path, "/") {
+		path = pf.Path[1:]
+	}
+	url := fmt.Sprintf("http://%s:%d/%s", host, pf.LocalPort, path)
 
 	// We panic on error here because we provide the URL format ourselves,
```

Comment:
This is no longer true! With this change, a user could put a bad string in their Tiltfile and cause Tilt to panic.

It might be simpler to make `pf.Path` a `URL` and assert `!url.IsAbs` at parse time.

Reply:
ughhhh you're right and i hate it.

are you suggesting IsAbs to check if it has a leading slash or to check that someone didn't pass in a whole-ass url with scheme etc?